roles/ceph-validate/tasks/main.yml

```diff
@@ -205,6 +205,13 @@
       fail:
         msg: "you must add at least one node in the [grafana-server] hosts group"
       when: groups[grafana_server_group_name] | length < 1
+
+    - name: fail when dashboard_admin_password and/or grafana_admin_password are not set
+      fail:
+        msg: "you must set dashboard_admin_password and grafana_admin_password."
```

Comment:
this is a problem for tripleo though, we don't set grafana_admin_password unless the dashboard is enabled ... so the check would interrupt the deployment

can we add a condition on the grafana_admin_password to run only if grafana_server_group is > 0 ?

Reply:
@gfidente there's a condition on `dashboard_enabled` for this task

Reply:
see https://github.com/ceph/ceph-ansible/blob/master/roles/ceph-validate/tasks/main.yml#L227